modules/hal_nordic/nrfx/nrfx_glue.h

```diff
@@ -315,24 +315,26 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
 /*------------------------------------------------------------------------------*/
 
 /** @brief Bitmask that defines DPPI channels that are reserved for use outside of the nrfx library. */
-#define NRFX_DPPI_CHANNELS_USED   (NRFX_PPI_CHANNELS_USED_BY_BT_CTLR |    \
-				   NRFX_PPI_CHANNELS_USED_BY_802154_DRV | \
-				   NRFX_PPI_CHANNELS_USED_BY_MPSL)
+#define NRFX_DPPI_CHANNELS_USED   (NRFX_PPI_CHANNELS_USED_BY_BT_CTLR      \
+				   | NRFX_PPI_CHANNELS_USED_BY_802154_DRV \
+				   | NRFX_PPI_CHANNELS_USED_BY_MPSL       \
```

Comment:
Such changes should not be done in a `noup` commit. Apply them upstream if it's worth it.

Reply:
Removed